installation/proc-config-dynamic-plugins-rhdh-operator.adoc

```diff
@@ -0,0 +1,70 @@
+[id="proc-config-dynamic-plugins-rhdh-operator_{context}"]
+= Configuring dynamic plugins with the {product} Operator
+
+You can store the configuration for dynamic plugins in a ConfigMap object that the Custom Resource can reference.
+
+.Example using the GitHub dynamic plugin
+In {ocp-short}, you can use the following example as a base template to create a ConfigMap named `dynamic-plugins-rhdh`:
+
+[source, yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic'
+        disabled: false
+        pluginConfig: {}
+----
+
+[NOTE]
+--
+If the `pluginConfig` field references environment variables, you must define the variables in your `secrets-rhdh` secret.
+--
+
+.Procedure
+. From the {ocp-short} web console, select the *ConfigMaps* tab.
+. Click *Create ConfigMap*.
+. From the *Create ConfigMap* page, select the *YAML view* option in *Configure via* and edit the file, if needed.
+. Click *Create*.
+. Go to the *Topology* tab.
+. Click on the overflow menu for the {product} instance that you want to use and select *Edit Backstage* to load the YAML view of the {product} instance.
++
+image::rhdh/operator-install-2.png[]
+
+
+. Add the `spec.application.dynamicPluginsConfigMapName` field to the Custom Resource. For example:
++
+[source,yaml]
+----
+spec:
+  application:
+    appConfig:
+      mountPath: /opt/app-root/src
+      configMaps:
+        - name: app-config-rhdh
+    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
+    extraEnvs:
+      secrets:
+        - name: secrets-rhdh
+    extraFiles:
+      mountPath: /opt/app-root/src
+    replicas: 1
+    route:
+      enabled: true
+  database:
+    enableLocalDb: true
+----
+. Click *Save*.
+. Navigate back to the *Topology* view and wait for the {product} pod to start.
```

Comment:
Here we use "*Topology* view" but it's "*Topology* tab" in other places - I added something for a future docs mtg agenda to discuss, but I think it would be good to standardize on view since OCP refers to it as this. E.g. see https://docs.openshift.com/container-platform/4.14/applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.html#odc-monitoring-your-application-metrics_monitoring-project-and-application-metrics-using-developer-perspective

Reply:
Agreed!

Reply:
https://issues.redhat.com/browse/RHIDP-2256